docs/docs/account/user-permissions.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: User Permissions
+description: GrowthBook's User Permissions
+sidebar_label: User Permissions
+slug: user-permissions
+---
+
+GrowthBook supports user roles and permissions at both the global and project level. This allows you to control who can access GrowthBook and what they can do.
+
+When inviting a new user to your organization, you can give the user a global role, which defines their permissions across all projects. If you are enrolled in a `Pro` or `Enterprise` account, you
+have the added ability to give a user project-level roles.
+
+:::note
+
+For instance, you can set a user's global role as `Collaborator`, which allows them to view features and experiements, as well as add
+comments and contribute ideas, and then give them an `Experimenter` role for a particular project, which allows them to create and run experiments, but only for the specified project.
+
+:::
+
+Below are the roles currently available in GrowthBook, ordered from least permissive to most. Each additional level inherits all of the permissions from the roles below it.
+
+- `readonly` - This role allows the user to view feature flags and experiments, but not make any changes.
+- `collaborator` - This role allows the user to view feature flags and experiments, as well as add comments and contribute ideas.
+- `engineer` - This role allows the user to manage feature flags, but not experiments.
+- `analyst` - This allows the user to analyze experiments.
+- `experimenter` - This allows the user to create feature flags and run experiments.
+- `admin` - This allows the user to manage all aspects of GrowthBook, including billing, organizational settings, and team members.
+
+And finally, there is one last level to GrowthBook's user permissions, which determines environment specific limits. This permission level only applies to `engineer` and `experimenter` roles in organizations in a `Pro` or `Enterprise` account. This
```

Comment:
This section could use a header or more clear context, maybe a relevant screenshot?

Reply:
Updated!